test/integration/ratio-Encounter-reuseObservationFunction/cql/ratio-Encounter-reuseObservationFunction.cql

```diff
@@ -0,0 +1,24 @@
+library RatioEncounterReuseObservation version '0.0.002'
+
+using FHIR version '4.0.1'
+
+include FHIRHelpers version '4.0.1'
+
+parameter "Measurement Period" Interval<DateTime>
+
+context Patient
+
+define "ipp":
+  ["Encounter"] E where E.period.start during "Measurement Period"
+ 
+define "denom":
+  ["Encounter"] E where E.period.end during "Measurement Period" and E.status = 'finished'
+
+define "numer":
+  "ipp"
+
+define function daysObs(e Encounter):
+  duration in days of e.period
+
+define function hoursObs(e Encounter):
+  duration in hours of e.period
```

Comment:
should we get rid of hoursObs in this specific fixture since it is not applicable to the test case?

Reply:
Oh true. Will remove.

Reply:
Removed in a771ed4.